sources/packages/web/src/components/common/sin/AddNewSIN.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <v-form ref="addNewSINForm">
+    <modal-dialog-base
+      title="Add a new SIN"
+      :showDialog="showDialog"
+      max-width="730"
+    >
+      <template #content>
+        <error-summary :errors="addNewSINForm.errors" />
+        <div class="pb-2">
+          <span class="label-value"
+            >Enter a new SIN for the student. The SIN will be validated with
+            Employment and Social Development Canada (ESDC) unless you skip the
+            validations.</span
+          >
+        </div>
+        <v-text-field
+          hide-details
```

Comment:
so, what decides if we `hide-details` or not?

Reply:
We can show @hellolynn-tbtb the difference and have a final decision on this.
@guru-aot can you do it, please?

Reply:
confirmed with @hellolynn-tbtb to have  hide-details="auto"